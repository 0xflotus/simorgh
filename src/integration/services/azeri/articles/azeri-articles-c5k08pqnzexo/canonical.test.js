/**
 * @service azeri
 * @pathname /azeri/articles/c5k08pqnzexo
 */

import runCanonicalTests from '../../../../pages/articles/canonicalTests';

describe('Canonical azeri articles', runCanonicalTests);
