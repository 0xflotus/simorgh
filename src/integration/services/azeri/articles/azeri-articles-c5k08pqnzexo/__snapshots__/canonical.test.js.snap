// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical azeri articles A11y I can see the skip to content link 1`] = `"Mətnə keçid"`;

exports[`Canonical azeri articles Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=6&amp;p=azeri.articles.c5k08pqnzexo.page&amp;x1=[urn:bbc:optimo:c5k08pqnzexo]&amp;x2=[responsive]&amp;x3=[news-azeri]&amp;x4=[az]&amp;x7=[article]&amp;x8=[simorgh-nojs]&amp;x9=[Bu+test+məqaləsinin+SEO+başlığı]&amp;x11=[2019-07-12T13:04:52.555Z]&amp;x12=[2019-09-27T15:35:50.759Z]\\"/>"`;

exports[`Canonical azeri articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/azeri"`;

exports[`Canonical azeri articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. BBC kənar saytların məzmununa məsul deyil. Bizim kənar keçidlərə dair yanaşmamız barədə oxuyun."`;

exports[`Canonical azeri articles Header I can see the branding 1`] = `"BBC News, Azərbaycanca"`;

exports[`Canonical azeri articles I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/d14e/test/2bb10ad0-e135-11e9-b23c-f1ac9644bb55.jpg"`;

exports[`Canonical azeri articles I can see an image with a caption 2`] = `"Şəklin alt yazısı, Bu test şəkli, müəllif hüquqları BBC, Fransanın bir xəritəsini göstərir. Şəkil ilk üç blokdadır və bu başlıq var."`;

exports[`Canonical azeri articles SEO Canonical link 1`] = `"http://localhost:7080/azeri/articles/c5k08pqnzexo"`;

exports[`Canonical azeri articles SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical azeri articles SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical azeri articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical azeri articles SEO Heading level 1 1`] = `"Bu test məqaləsinin başlığı budur"`;

exports[`Canonical azeri articles SEO Lang attribute 1`] = `"az"`;

exports[`Canonical azeri articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png",
      "width": 1024,
    },
    "name": "BBC News Azərbaycanca",
    "noBylinesPolicy": "https://www.bbc.com/azeri/institutional-49283479#authorexpertise",
  },
  "dateModified": "2019-09-27T15:35:50.759Z",
  "datePublished": "2019-07-12T13:04:52.555Z",
  "headline": "Bu test məqaləsinin SEO başlığı",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/azeri/articles/c5k08pqnzexo",
    "@type": "WebPage",
    "name": "Bu test məqaləsinin SEO başlığı",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png",
      "width": 1024,
    },
    "name": "BBC News Azərbaycanca",
    "publishingPrinciples": "https://www.bbc.com/azeri/institutional-49283479",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png",
  "url": "http://localhost:7080/azeri/articles/c5k08pqnzexo",
}
`;

exports[`Canonical azeri articles SEO OG description 1`] = `"Bu test məqaləsinin SEO başlığı"`;

exports[`Canonical azeri articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png"`;

exports[`Canonical azeri articles SEO OG image alt 1`] = `"BBC News Azərbaycanca"`;

exports[`Canonical azeri articles SEO OG locale 1`] = `"az-AZ"`;

exports[`Canonical azeri articles SEO OG site name 1`] = `"BBC News Azərbaycanca"`;

exports[`Canonical azeri articles SEO OG title 1`] = `"Bu test məqaləsinin SEO başlığı - BBC News Azərbaycanca"`;

exports[`Canonical azeri articles SEO OG type 1`] = `"article"`;

exports[`Canonical azeri articles SEO OG url 1`] = `"http://localhost:7080/azeri/articles/c5k08pqnzexo"`;

exports[`Canonical azeri articles SEO Page title 1`] = `"Bu test məqaləsinin SEO başlığı - BBC News Azərbaycanca"`;

exports[`Canonical azeri articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical azeri articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical azeri articles SEO Twitter creator 1`] = `"@bbcazeri"`;

exports[`Canonical azeri articles SEO Twitter description 1`] = `"Bu test məqaləsinin SEO başlığı"`;

exports[`Canonical azeri articles SEO Twitter image alt 1`] = `"BBC News Azərbaycanca"`;

exports[`Canonical azeri articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/azeri.png"`;

exports[`Canonical azeri articles SEO Twitter site 1`] = `"@bbcazeri"`;

exports[`Canonical azeri articles SEO Twitter title 1`] = `"Bu test məqaləsinin SEO başlığı - BBC News Azərbaycanca"`;
